framework/src/ble_manager/ble_manager_state.c

```diff
@@ -466,7 +466,7 @@ ble_result_e blemgr_handle_request(blemgr_msg_s *msg)
 		2. auto connection
 		*/
 		ret = TRBLE_SUCCESS;
-		ble_client_ctx *ctx = (ble_client_ctx *)msg->param;
+		ble_client_ctx_internal *ctx = (ble_client_ctx_internal *)msg->param;
```

Comment:
you should check validation(null check)of msg & param right before switch-case, not here

Reply:
'msg' is guaranteed that it is not NULL.
It is always created in ble_msg_handler.